Makefile

```diff
@@ -164,7 +164,7 @@ test-e2e: generate fmt vet manifests test-validation test-functional  ## Run e2e
 
 .PHONY: build
 build: prereqs generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+	CGO_ENABLED=1 go build -o bin/manager main.go
```

Comment:
Same question as before: why do the controllers need libxdp/libbpf when only the daemon actually manages attachments / detachments?